src/status_im2/contexts/chat/photo_selector/album_selector/view.cljs

```diff
@@ -61,15 +61,28 @@
 
 (defn- f-album-selector
   [{:keys [scroll-enabled on-scroll]} album? selected-album top]
-  (let [albums          (rf/sub [:camera-roll/albums])
-        albums-sections [{:title no-title :data (:smart-albums albums)}
-                         {:title (i18n/label :t/my-albums) :data (:my-albums albums)}]
-        window-height   (:height (rn/get-window))]
+  (let [albums                           (rf/sub [:camera-roll/albums])
+        add-recent-count-to-recent-album (-> albums
```

Comment:
this should be inside subscription `:camera-roll/albums-with-count`

Reply:
or in `:camera-roll/albums`

Reply:
I went with a new subscription of :camera-roll/total-photos-count